rapid_response_xblock/static/js/src/rapid.js

```diff
@@ -503,8 +552,16 @@
         pollMinutes = Math.floor(totalSeconds / 60);
       }
       $timer.text(pollMinutes + "m : " + pollSeconds + "s");
+      $timer.toggleClass("on", state.is_open);
       $timerSpinner.toggleClass('hidden', !state.is_open);
       $timerSpinnerText.toggleClass('hidden', !state.is_open);
+      $numStudents.toggleClass('hidden', !state.is_open);
+
+      // using chartIndex 0 since this message should only get shown if there is only one chart visible
+      if (state.runs.length > 0) {
```

Comment:
Nit: You can probably get rid of the ternary operator below and just add it to this condition `if (state.runs.length > 0 && state.is_open)`. The element is already being hidden if the problem isn't open. You could move the `toggleClass` below this conditional so we don't show the element before the correct text is set. Won't hold up the PR for this, though.